scripts/populate-worksection.js

```diff
@@ -0,0 +1,277 @@
+  featuredImage.src = pro.featuredImage;
+  featuredImage.alt = `${pro.id} picture`;
+  featuredImage.classList.add('featuredImage-mobile');
+  featuredDimageDiv.appendChild(featuredImage);
+  projectDiv.appendChild(featuredDimageDiv);
+
+  /* project description (details) */
+  const projectDetailDiv = document.createElement('div');
+  const name = document.createElement('h2');
+  name.innerHTML = pro.name;
+  name.classList.add('project-title');
+  projectDetailDiv.appendChild(name);
+  projectDiv.appendChild(projectDetailDiv);
+
+  /* list of languages */
+  const subTitle = makeUL(pro.sub);
+  subTitle.classList.add('icons');
+  projectDetailDiv.appendChild(subTitle);
+
+  const description = document.createElement('p');
+  description.innerHTML = pro.description;
+  projectDetailDiv.appendChild(description);
+
+  const languagesList = makeL(pro.langs);
+  languagesList.classList.add('icons');
+  projectDetailDiv.appendChild(languagesList);
+
+  const more = document.createElement('button');
+  more.id = pro.id;
+  more.type = 'button';
+  more.innerHTML = 'See Project';
+  more.classList.add('see-more-button');
+
+  projectDetailDiv.appendChild(more);
+  portfolio.appendChild(projectDiv);
+}
+
+function addPortfolio() {
+  for (let i = 0; i < projects.length; i += 1) {
+    addWorkSection(projects[i]);
+  }
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  addPortfolio();
+}, false);
+
+// pup up
+const projectButtons = [];
+const buttons = document.querySelectorAll('button');
+const modal = document.getElementById('myModal');
+
+buttons.forEach((button) => {
+  const [btn] = button.id.split('-');
+  if (btn === 'project') projectButtons.push(button);
+});
+
+const displayModal = (id) => {
+  let li = '';
+  const project = projects.find((p) => p.id === id);
+  project.langs.forEach((project) => {
+    li += `<li class="languages">${project}</li>`;
+  });
+
+  const render = `
+  <div class="modal-content">
+  <span class="close">&times;</span>
+  <h2 class="modal-card-title">${project.name}</h2>
+  <ul class="icons">
+      <li class="canopy">CANOPY</li>
+      <li class="counter"><span class='dot'> <span>
+      </li>
+      <li class="bg">Back End Dev</li>
+      <li class="counter"><span class='dot'> <span>
+      </li>
+      <li class="bg">2015</li>
+  </ul>
+  <div>
+    <img src=${project.featuredImage} alt="featured project image" class="modal-card-img">
+  </div>
+  <div class="modal-project-description">
+      <p class="modal-description">${detailText}</p>
+      <div class="modal-links">
+          <ul class="icons">
+            ${li}
+            <li class="languages">Github </li>
+            <li class="languages">Ruby </li>
+            <li class="languages"> Bootsrap</li>
+          </ul>
+          <hr>
+          <div class="live-links">
+              <button class="btn-enabled" id='live-link' type="button">
+              
+                  see live
+                  <img src="./images/live-icon.png">
```

Comment:
- [ ] Please make sure you implement the requests of the previous reviewer. The image is still not loading because, it's still referenced incorrectly. Please use the exact path to the file including its capitalization.

Reply:
thank you very much for the nice review. I doubled checked it and the img's url and it displayes the image correctly on my local machine. I dont know why it is not showing to you.  
![image](https://user-images.githubusercontent.com/97679679/180617887-1ca3b7f0-d07b-48e2-8c50-8dbd5bc243f3.png)